microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md

```diff
@@ -0,0 +1,171 @@
+
+
+<!-- ====================================================================== -->
+## Accessibility improvements for the CSS Overview tool
+
+<!-- Subtitle: The CSS Overview tool is now easier to use with assistive technology such as screen readers. -->
+
+In Microsoft Edge 112, the **CSS Overview** tool was updated to display a list of non-simple selectors when taking an overview snapshot of a webpage's CSS.  In Microsoft Edge 114, the **Non-simple selectors** section of the **CSS Overview** tool is now easier to use with assistive technology, such as screen readers.
+
+When you click a selector, or navigate to a selector and then press **Enter**, screen readers now announce "Copied CSS selector":
+
+![Navigating the CSS Overview tool with the keyboard](./devtools-114-images/non-simple-selectors-css-overview-a11y.png)
+
+See also:
+* [CSS Overview tool displays non-simple selectors for quick performance gains](../04/devtools-112.md#css-overview-tool-displays-non-simple-selectors-for-quick-performance-gains) in _What's New in DevTools (Microsoft Edge 112)_.
+* [Non-simple selectors](../../../css/css-overview-tool.md#non-simple-selectors) in _Optimize CSS styles with the CSS Overview tool_.
+* [Navigate DevTools with assistive technology](../../../accessibility/navigation.md)
+
+
+<!-- ====================================================================== -->
+## DevTools extension for VS Code has better support for high contrast mode
+
+<!-- Subtitle: Hovering over icons in high contrast mode in the VS Code extension now renders with sufficient contrast. -->
+
+In previous versions of the Microsoft Edge DevTools extension for Visual Studio Code, in high contrast mode, hovering over icons within the tools didn't render the UI controls with sufficient contrast.  This issue has been fixed.
+
+For example, in the **Styles** pane in the **Elements** tool, checkboxes for applying styles to elements now render correctly in high contrast mode:
+
+![Hovering over checkboxes in the Styles pane in high contrast mode in the VS Code extension](./devtools-114-images/vs-code-hc-mode-styles-pane.png)
+
+Also, in the **Console** tool, icons such as **Show console sidebar** now render correctly in high contrast mode:
+
+![Hovering over the Show console sidebar button in high contrast mode in the VS Code extension](./devtools-114-images/vs-code-hc-mode-console-tool.png)
+
+See also:
+* [Microsoft Edge DevTools extension for Visual Studio Code](../../../../visual-studio-code/microsoft-edge-devtools-extension.md)
+* [Turn high contrast mode on or off in Windows](https://support.microsoft.com/windows/turn-high-contrast-mode-on-or-off-in-windows-909e9d89-a0f9-a3a9-b993-7a6dcee85025)
+
+
+<!-- ====================================================================== -->
+## Announcements from the Chromium project
+
+Microsoft Edge 114 also includes the following updates from the Chromium project:
```

Comment:
keep these announcements only:

[Debug Autofill using the Elements panel and Issues tab](https://developer.chrome.com/blog/new-in-devtools-114/#autofill)
[Lighthouse 10.1.1](https://developer.chrome.com/blog/new-in-devtools-114/#lighthouse)
[Performance enhancements](https://developer.chrome.com/blog/new-in-devtools-114/#performance)
[performance.mark() shows timing on hover in Performance > Timings](https://developer.chrome.com/blog/new-in-devtools-114/#mark)
[profile() command populates Performance > Main](https://developer.chrome.com/blog/new-in-devtools-114/#profile)
[Warning for slow user interactions](https://developer.chrome.com/blog/new-in-devtools-114/#slow-interaction-warning)
[JavaScript Profiler deprecation: Phase three](https://developer.chrome.com/blog/new-in-devtools-114/#js-profiler)